app/models/site.rb

```diff
@@ -0,0 +1,10 @@
+class Site < ApplicationRecord
+  validates :name, presence: true
+  validates :address, presence: true
+  validates :region, presence: true
+  validates :longitude, presence: true
+  validates :latitude, presence: true
+
+  belongs_to :diver
+  belongs_to :dive 
+end
```

Comment:
we likely need to re-evalute your er diagram. a site should have many dives, which implicitly links a diver to a site. unless of course a diver has favorited sites, but the ER diagram would still need to change